ballerina/tests/test.bal

```diff
@@ -20,19 +20,121 @@ configurable string clientId = ?;
 configurable string clientSecret = ?;
 configurable string subDomain = ?;
 
+Client mcClient = check new (
+    subDomain,
+    config = {
+        auth: {
+            clientId,
+            clientSecret
+        }
+    }
+);
+
 @test:Config {
+    enable: false
 }
 public function testGetJourneys() returns error? {
-    Client mcClient = check new (
-        subDomain,
-        config = {
-            auth: {
-                clientId,
-                clientSecret
-            }
-        }
-    );
-
     JourneysList response = check mcClient->getJourneys();
     test:assertNotEquals(response.items, [], msg = "Expected non-empty journeys list, but got empty list");
 }
+
+@test:Config {
+}
```

Comment:
Can simply do this?

```suggestion
@test:Config
```

Reply:
Fixed in [5343047](https://github.com/ballerina-platform/module-ballerinax-salesforce.marketingcloud/pull/3/commits/53430471cf17cccfb5d1b076b987189e169c9977)